tpcx_bb/cluster_configuration/cluster-startup.sh

```diff
@@ -1,25 +1,22 @@
 #IB, NVLINK, or TCP
-CLUSTER_MODE=$1
+ROLE=$1
+CLUSTER_MODE="NVLINK"
 USERNAME=$(whoami)
+TPCX_BB_HOME=$HOME/tpcx-bb/tpcx_bb
 
 MAX_SYSTEM_MEMORY=$(free -m | awk '/^Mem:/{print $2}')M
 DEVICE_MEMORY_LIMIT="25GB"
 POOL_SIZE="30GB"
 
-# Fill in your environment name and conda path on each node
-TPCX_BB_HOME="/home/$USERNAME/shared/tpcx-bb"
 CONDA_ENV_NAME="rapids-tpcx-bb"
-CONDA_ENV_PATH="/home/$USERNAME/conda/etc/profile.d/conda.sh"
-
-# TODO: Unify interface/IP setting/getting for cluster startup
-# and scheduler file
-INTERFACE="ib0"
+CONDA_ENV_PATH="/conda/etc/profile.d/conda.sh"
```

Comment:
Can probably back out this change by building the Docker image such that conda is installed to `/home/root/conda`.

Reply:
I think it's probably fine as is